lib/pseudolocalization.rb

```diff
@@ -5,9 +5,12 @@ module Pseudolocalization
   module I18n
     class Backend
       attr_reader :original_backend
+      attr_accessor :ignores
 
       def initialize(original_backend)
         @original_backend = original_backend
+        @ignores = []
```

Comment:
Could potentially pass as a second `initialize` param but given ignores are optional, this is cleaner.

Reply:
Yeah, at this point, we only have one configuration param (this one) so i didn't feel like mucking it all up was worth it